roles/dosomething.phoenix/templates/nginx/app.j2

```diff
@@ -30,6 +30,7 @@ server {
   location / {
     # This is cool because no php is touched for static content
     try_files $uri @rewrite;
+    error_page 404 /leftathetrafficlight.php;
```

Comment:
Yes, I see. But please leave a comment in the config file explaining why it's there.